core/iwasm/compilation/aot_llvm.c

```diff
@@ -2444,3 +2475,65 @@ aot_get_func_from_table(const AOTCompContext *comp_ctx, LLVMValueRef base,
 fail:
     return NULL;
 }
+
+LLVMValueRef
+aot_load_const_from_table(AOTCompContext *comp_ctx, LLVMValueRef base,
+                          const WASMValue *value, uint8 value_type)
+{
+    LLVMValueRef const_index, const_addr, const_value;
+    LLVMTypeRef const_ptr_type;
+    char buf[128] = { 0 };
+    int32 index;
+
+    switch (value_type) {
+        case VALUE_TYPE_I32:
+            snprintf(buf, sizeof(buf), "i32#%08X", value->i32);
+            const_ptr_type = INT32_PTR_TYPE;
+            break;
+        case VALUE_TYPE_I64:
+            snprintf(buf, sizeof(buf), "i64#%016" PRIx64, value->i64);
+            const_ptr_type = INT64_PTR_TYPE;
+            break;
+        case VALUE_TYPE_F32:
+            snprintf(buf, sizeof(buf), "i32#%08X", value->i32);
```

Comment:
```suggestion
            snprintf(buf, sizeof(buf), "i32#%08X", value->f32);
```
Seems there should be f32?

Reply:
f32#%08X?

Reply:
Should be i32, store the raw int bits to make it simple and avoid data loss (possible data loss when using f32, e.g. NaN values).